1차/이동건_체육복.py

```diff
@@ -0,0 +1,29 @@
+# 21:35 ~ 21:53
+
+def to_dict(num):
+    data = {}
+    for i in range(1, num + 1):
+        data.update({i:0})
+    return data
+
+def solution(n, lost, reserve):
+    students = to_dict(n)
```

Comment:
굳이 function으로 뺄 이유가 없어보입니다.
dict 선언후 update function을 사용하기때문에 아마 시간 소요가 좀더 있을것으로 보이기도 하구요.

```suggestion
def solution(n, lost, reserve):
    students = { i :0 for i in range(1,n+1) }
```
으로 한줄로 바꿀수있습니다.